src/SignalR/clients/ts/FunctionalTests/yarn.lock

```diff
@@ -43,21 +43,6 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-9.6.49.tgz#ab4df6e505db088882c8ce5417ae0bc8cbb7a8a6"
   integrity sha512-YY0Okyn4QXC4ugJI+Kng5iWjK8A6eIHiQVaGIhJkyn0YL6Iqo0E0tBC8BuhvYcBK87vykBijM5FtMnCqaa5anA==
 
-"@types/strip-bom@^3.0.0":
```

Comment:
I assume this is expected?

Reply:
Are you referring to the file changing? Yes, I updated dependencies so this file gets updated.